internal/csi-addons/rbd/reclaimspace.go

```diff
@@ -90,12 +98,13 @@ func (rscs *ReclaimSpaceControllerServer) ControllerReclaimSpace(
 // of CSI-addons reclaimspace controller service spec.
 type ReclaimSpaceNodeServer struct {
 	*rs.UnimplementedReclaimSpaceNodeServer
+	*util.VolumeLocks
```

Comment:
same here, don't inherit the functions of the struct, just add a new member for it.